### PR TITLE
[V2-2986] cli chapter element tutorial

### PR DESCRIPTION
**Pull request by santios-volusion**
This includes a chapter on the current behavior of the CLI, and some words on how to use it properly.

### CODE REVIEW

File: cli/README.md [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+
+> Pro Tip: We recommend to checkout your repo and point your code to the commit `v1 released`
+
+You can then make changes, and start fresh, from the `v1 released` commit state.
+
+### Creating a Major release
+
+Maybe you are adding new features to your block, the block configuration is not compatible with the previous, and/or the look and feel of the block changed. Those are all good reasons to create a major release.
+
+In order to do that, we recommend to create a new branch, where you will keep the `version 2` of the code:
```
Comment (jason-covington) [inline cli/README.md]: "we recommend creating a new branch,"

File: cli/README.md [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+
+```bash
+git checkout -b version2
+```
+
+Then, run the following `publish` command with the `-m` flag:
```
Comment (jason-covington) [inline cli/README.md]: What does the "-m" flag do by itself? Or should there be a commit message?

File: cli/README.md [hunk 3]
```diff
@@ -0,0 +1,377 @@
...
+
+# Make a minor change to v1
+git checkout master
+
+#make changes
+npm run build
+element update
+element release -n "v1.2 released"
+
+# As always, remember to save the relaese as a commit
```
Comment (jason-covington) [inline cli/README.md]: "release"

**Review by jason-covington — CHANGES_REQUESTED** [review]

File: cli/README.md [hunk 4]
```diff
@@ -0,0 +1,377 @@
...
+element new -h
+```
+
+Let's create our new block:
+
+```bash
+element new MyFirstBlock
+```
+
+> Pro Tip: Use camel case or dash separated words as your block folder name
```
[the remark this answers is not in the document]
Comment (santios-volusion) [reply cli/README.md]: Required, I think. Gonna remove the top, and use bold.

File: cli/README.md (hunk 2 above)
Comment (santios-volusion) [reply cli/README.md]: The -m is not related to git. Is just `m` from `major`

**Comment by santios-volusion** [comment]
@jason-covington Thanks for your feedback. Updated.

**Review by jason-covington — APPROVED** [review]
👍

**Pull request merged**